app/services/katello/pulp3/module_stream.rb

```diff
@@ -0,0 +1,47 @@
+module Katello
+  module Pulp3
+    class ModuleStream < PulpContentUnit
+      include LazyAccessor
+
+      def self.content_api
+        PulpRpmClient::ContentModulemdApi.new(Katello::Pulp3::Repository::Yum.api_client(SmartProxy.pulp_master!))
+      end
+
+      def self.ids_for_repository(repo_id)
+        repo = Katello::Pulp3::Repository::ModuleStream.new(Katello::Repository.find(repo_id), SmartProxy.pulp_master)
+        repo_content_list = repo.content_list
```

Comment:
Do these exist? Katello::Pulp3::Repository::ModuleStream , Katello::Pulp3::Repository::YumMetadataFile, Katello::Pulp3::Repository::Erratum, Katello::Pulp3::Repository::Rpm .

Reply:
oh oops, that's a mistake!